src/Nokia_LCD.h

```diff
@@ -275,6 +296,9 @@ class Nokia_LCD {
      */
     bool printCharacter(char character);
 
+protected:
+    bool coupled = false;
+private:
```

Comment:
Should be `private` and use the `m` prefix, i.e. `mCoupled`.
```suggestion
    bool mCoupled = false;
```